src/model/writer.js

```diff
@@ -1141,3 +1174,10 @@ function isSameTree( rootA, rootB ) {
 
 	return false;
 }
+
+/**
+ * @typedef {Object} SplitResult
```

Comment:
Do we need to define a typedef here? I think we can describe the return object inline, in the `split` documentation.